src/shared/utilities/pathUtils.ts

```diff
@@ -113,3 +113,12 @@ export function getDriveLetter(path: string): string {
 
     return fullpath.substring(0, 1)
 }
+
+/**
+ * Returns the name of a file without an extension
+ * @param file File name from which the extension is removed
+ * @returns A string file name with the extension removed
+ */
+export function trimExtension(file: string) {
+    return file.replace(/\.[^/.]+$/, '')
+}
```

Comment:
If we are adding it as util function here, can you add comment about how files with multiple extensions are handled? (e.g., `a.tar.gz`)

Reply:
Good point, `In the case of multiple extensions, just trims the last one. Eg 'a.tar.gz' -> 'a.tar'`

Reply:
ok, just make sure when other people read it, they know this behavior